dataactvalidator/health_check.py

```diff
@@ -4,6 +4,13 @@
 from dataactcore.logging import configure_logging
 from dataactcore.utils.jsonResponse import JsonResponse
 
+# Datadog import
+USE_DATADOG = False
```

Comment:
Add a comment here to let developers know that this value is being changed with Ansible, so nobody deletes it thinking the next `if` can never be reached.